ibc-primitives/src/types/timestamp.rs

```diff
@@ -284,9 +301,8 @@ pub enum TimestampError {
 impl std::error::Error for TimestampError {
     fn source(&self) -> Option<&(dyn std::error::Error + 'static)> {
         match &self {
-            Self::ParseInt(e) => Some(e),
-            Self::TryFromInt(e) => Some(e),
-            Self::Conversion(e) => Some(e),
```

Comment:
why is `FailedToConvert` case removed ?

Reply:
[4f11258](https://github.com/cosmos/ibc-rs/pull/1353/commits/4f112581c4155194e8830db65f85e14355e6db03)